docs/oauth-2.0.md

```diff
@@ -47,12 +47,13 @@ For instructions on setting up your credentials properly, see the
 already have an access token, you can make a request in the following way:
 
 ```java
-GoogleCredential credential = new GoogleCredential().setAccessToken(accessToken);
-Plus plus = new Plus.builder(new NetHttpTransport(),
-                             JacksonFactory.getDefaultInstance(),
-                             credential)
-    .setApplicationName("Google-PlusSample/1.0")
-    .build();
+  GoogleCredentials googleCredentials = GoogleCredentials.create(ACCESS_TOKEN);
+    HttpRequestInitializer requestInitializer = new HttpCredentialsAdapter(googleCredentials);
+    Storage storage = new Storage.Builder(HTTP_TRANSPORT,
+        JSON_FACTORY,
```

Comment:
Can you restore these values to the non-constants? These are not defined in this sample code.